doc/source/whatsnew/v0.19.0.txt

```diff
@@ -330,6 +330,7 @@ API changes
 ~~~~~~~~~~~
 
 
+- ``Series.to_csv`` has dropped the ``nanRep`` parameter in favor of ``na_rep`` (:issue:`13804`)
```

Comment:
It had not really been dropped 'in favor of', I think? As it did actually not have any effect?